FSI/DisplacementDelta.C

```diff
@@ -28,6 +28,10 @@ void preciceAdapter::FSI::DisplacementDelta::write(double * buffer)
     FatalErrorInFunction
         << "Writing displacementDeltas is not supported."
         << exit(FatalError);
+
+    if(provideMeshConnectivity)
+        Info<<"Warning: Mesh connectivity is not supported for FSI cases"<<endl;
```

Comment:
@MakisH I guess we could use here some OpenFOAM specific warning function with `std::cerr` type

Reply:
See comment, below: We can use the Debug Info for this purpose.